linux_os/guide/services/sssd/sssd-ldap/sssd_ldap_configure_tls_ca_dir/tests/ldap_tls_cacertdir_bad_value_conf_d.fail.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# packages = /usr/lib/systemd/system/sssd.service
+# profiles = xccdf_org.ssgproject.content_profile_stig
```

Comment:
I don't see a reason for having the profiles key here. Why do you add it?

Reply:
Just copied it from the other tests